main.js

```diff
@@ -445,4 +468,15 @@ function hideThemeCenter() {
   document.getElementById('command-center').classList.remove('hidden');
 }
 
-
+function betterSplit(str) {
+	// Regex tester for:
+		// Normal a-z, A-Z
+		// punjabi,
+		// CJK,
+		// Russian
+	// TODO: Add support for remaining European languages
+	// Cleaner way to do this???
+	const regex = /[\u0A00-\u0A7F\u3040-\u30ff\u3400-\u4dbf\u4e00-\u9fff\uf900-\ufaff\uff66-\uff9f\u3131-\uD79D]+|[\u0400-\u04FFa-zA-Z0-9!:;'",.\/?!@#$%^&*()_}{\[\]-]+\'*[a-z]*/g;	
+	let array = [...str.match(regex)];
+  return array;
+}
```

Comment:
Make sure you have a newline at the end of your file.

Reply:
Please do not "reformat" the whole file! Configure your editor to use the original code style.

Reply:
Yh, I'll fix that, while changing the tab indents to spaces I must've inadvertently reformated the entire file.